server-spi-private/src/main/java/org/keycloak/migration/MigrationModelManager.java

```diff
@@ -100,7 +100,7 @@
     public static void migrate(KeycloakSession session) {
         session.setAttribute(Constants.STORAGE_BATCH_ENABLED, Boolean.getBoolean("keycloak.migration.batch-enabled"));
         session.setAttribute(Constants.STORAGE_BATCH_SIZE, Integer.getInteger("keycloak.migration.batch-size"));
-        MigrationModel model = session.getProvider(ServerInfoProvider.class).getMigrationModel();
+        MigrationModel model = session.getProvider(ServerInfoProvider.class, "jpa").getMigrationModel();
```

Comment:
Why is this change here?

Reply:
I forgot to revert this. It's from my local tests, reverting, thank you.